active/Breadcrumb/Breadcrumb.md

```diff
@@ -0,0 +1,150 @@
+reason it's necessary for a caller to catch an exception from an API, call that
+out with an explanation either here or in the Examples -->
+
+# API Notes
+<!-- Option 1: Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. For properties, specify the default value of the property if it
+isn't the type's default (for example an int-typed property that doesn't default to zero.) -->
+
+| Name | Description | Default |
+| :---------- | :------- | :------- |
+| ItemsSource | provides the content of the Breadcrumb | Null |
+| ItemClicked | Raised when a user interaction causes a jump to the CurrentItem.  | N/A |
+
+Note that V2 of Breadcrumb will add flyouts from chevrons to view children of a node. Additional properties will be added to the API to enable this functionality. 
+
+<!-- Option 2: Put these descriptions in the below API Details section,
+with a "///" comment above the member or type. -->
+
+# API Details
+<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+
+## API
+~~~~
+
+runtimeclass BreadcrumbItemClickedEventArgs
+{
+    Object Item { get; };
+}
+
+unsealed runtimeclass BreadcrumbItem : Windows.UI.Xaml.Controls.ContentControl
+{
+    BreadcrumbItem();
+}
+
+unsealed runtimeclass BreadcrumbLayout : NonVirtualizingLayout
+{
+    BreadcrumbLayout();
+}
+
+[MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+[MUX_PROPERTY_CHANGED_CALLBACK_METHODNAME("OnPropertyChanged")]
+unsealed runtimeclass Breadcrumb : Windows.UI.Xaml.Controls.Control
+{
+    Breadcrumb();
+
+    Object DropdownItemTemplate{ get; set; };
+    Object ItemsSource{ get; set; };
+    Object ItemTemplate{ get; set; };
+
+    event Windows.Foundation.TypedEventHandler<Breadcrumb, BreadcrumbItemClickedEventArgs> ItemClicked;
+
+    static Windows.UI.Xaml.DependencyProperty DropdownItemTemplateProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty ItemsSourceProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty ItemTemplateProperty{ get; };
+}
+
+ ~~~~
+
+<!-- # Appendix -->
+<!-- Anything else that you want to write down for posterity, but 
+that isn't necessary to understand the purpose and usage of the API.
+For example, implementation details. -->
+
+# Inputs and Accessibility
+## UI Automation Pattern
+Breadcrumb will use the [InvokePattern](!https://docs.microsoft.com/en-us/dotnet/framework/ui-automation/implementing-the-ui-automation-invoke-control-pattern).
+
+## Keyboard
+* The user can tab into the Breadcrumb and tab out. The first tab in gives focus to the first (highest-level) node. 
+Once the Breadcrumb has focus, the user can use the left/right arrows to navigate from node to node. 
+* If the Breadcrumb is crumbled, down/up will open the flyout from the ellipsis and down/up will navigate between the flyout items
+
+## Gamepad
+* Spatial navigation to focus and select the nodes.
```

Comment:
What does this mean?

Reply:
I believe this description can be a little longer and similar to the pips pager control

The Breadcrumb control can be navigated to via spatial navigation. The first visible breadcrumb item will automatically take focus and the items will be selected when navigated to and focused.

Left and right spatial navigation will focus and select the breadcrumb items. The Breadcrumb control will be navigated out of when the user navigates left or right when the user attempts to navigate past the first visible or last item. 

Down spatial navigation when the ellipsis item is focused will open the flyout containing the 'crumbled' items and the first item in the flyout will gain focus.

Reply:
It's easiest to understand Gamepad behavior by describing where it's different than normal keyboard focus. It sounds like the only difference is that you open the dropdown using Gamepad with Down, and using Keyboard with space bar?

Reply:
If the dropdowns are opened with Down, does that mean IsFocusEngagmentEnabled = True ? https://docs.microsoft.com/en-us/windows/uwp/design/input/gamepad-and-remote-interactions#focus-engagement

Reply:
discussion: A button should invoke the flyout, not Down